module/VuFind/src/VuFind/Controller/UpgradeController.php

```diff
@@ -849,6 +851,12 @@ public function homeAction()
             return $this->forwardTo('Upgrade', 'EstablishVersions');
         }
 
+        // Check for critical upgrades
+        $forward = $this->criticalCheckForBlowfishEncryption();
```

Comment:
Rather than directly calling this from homeAction, maybe it would be worth wrapping it in a `performCriticalChecks` method, so if we have additional critical checks in the future, there's an obvious place to add them -- it keeps the homeAction a bit more generic. Of course, since there's only one check at this point, we could also choose to do that refactoring later. I don't have strong feelings about what to do now, I just think that's the eventual place we should land if we end up with multiple critical checks.